internal/v2/fakes2av2/fakes2av2.go

```diff
@@ -96,7 +96,12 @@ func offloadPrivateKeyOperation(req *s2av2pb.OffloadPrivateKeyOperationReq, host
 			}
 		default:
 			s := fmt.Sprintf("Invalid hostname tied to SessionReq: %s", hostname)
-			return nil, errors.New(s)
+			return &s2av2pb.SessionResp {
+				Status: &s2av2pb.Status {
+					Code: 3,
+					Details: s,
+				},
```

Comment:
Let's use grpc.Code named variables.

Reply:
Will do! I'll include this when I merge PR#48 which has the change to use gRPC code named constants.